### PR TITLE
Upgrade admob extension

### PR DESCRIPTION
**Pull request by ClementPasteau**

### CODE REVIEW

File: Extensions/AdMob/admobtools.ts [hunk 1]
```diff
@@ -93,288 +172,415 @@ namespace gdjs {
...
+      interstitial.on('showfail', () => {
+        interstitialShowing = false;
+        interstitialErrored = true;
+      });
+      interstitial.on('dismiss', () => {
+        interstitialShowing = false;
+      });
+
+      try {
+        logger.info('loading Admob interstitial.....');
```
Comment (4ian) [inline Extensions/AdMob/admobtools.ts:313]: Is this for debugging? If not it should be Capitalized + single period.

Comment (ClementPasteau) [reply Extensions/AdMob/admobtools.ts:313]: I'll update all of them, I think it's valuable to keep the logs

File: Extensions/AdMob/admobtools.ts [hunk 2]
```diff
@@ -13,31 +49,100 @@ namespace gdjs {
...
+    document.addEventListener(
+      'deviceready',
+      async () => {
+        // Obtain user consent ?
+
+        await admob.start();
+
+        logger.info('AdMob succesfully started');
+
+        admobStarted = true;
```
Comment (4ian) [inline Extensions/AdMob/admobtools.ts]:
```suggestion
        logger.info('AdMob successfully started');
        admobStarted = true;
```

File: Extensions/AdMob/admobtools.ts [hunk 3]
```diff
@@ -93,288 +172,415 @@ namespace gdjs {
...
+        adUnitId,
+        position: atTop ? 'top' : 'bottom',
+        size: bannerRequestedAdSizeType,
+      });
+
+      banner.on('load', () => {
+        bannerShowing = true;
+        bannerLoaded = true;
+      });
+      banner.on('load_fail', () => {
```
Comment (4ian) [inline Extensions/AdMob/admobtools.ts:319]: 🚨 This should be 
```suggestion
      banner.on('loadfail', () => {
```

according to https://admob-plus.github.io/docs/cordova/ads/banner#loadfail and https://github.com/admob-plus/admob-plus/search?q=loadfail

Comment (ClementPasteau) [reply Extensions/AdMob/admobtools.ts:319]: oh yes forgot to update this one, nice catch

**Review by 4ian — APPROVED** [review]

**Pull request merged**